lib/postprocessing/image.js

```diff
@@ -32,14 +32,63 @@ const SUPPORTED_OUTPUT_FMT = {
 };
 
 // list of supported input formats (intermediate rendition)
-// in imagemagick format https://imagemagick.org/script/formats.php
-const SUPPORTED_INPUT_FMT = new Set([
-    "PNG",
-    "JPEG",
-    "TIFF",
-    "GIF",
-    "BMP"
-]);
+// maps imagemagick format ('PNG') https://imagemagick.org/script/formats.php to our fmt ('png')
+// Note: order is important and defines priority for intermediate format - first entry will be most preferred
+const SUPPORTED_INPUT_TYPES = {
+    "TIFF": "tif",  // most preferred format as it allows to preserve many features
```

Comment:
> Because if you loop over an object's keys, I think it's in alphanumerical order.

No, the order is actually preserved: https://www.stefanjudis.com/today-i-learned/property-order-is-predictable-in-javascript-objects-since-es2015/

It makes it a bit easier since I need an ordered map here (and it allows to use native features to access it). It could be switched to an array with "pair" objects in it in case the order is not guaranteed to be stable.

Reply:
Oh, interesting! 
I still think it is a bit fragile (see https://stackoverflow.com/questions/30076219/does-es6-introduce-a-well-defined-order-of-enumeration-for-object-properties), though, since business logic depends on this being respected.